neon_data_models/models/base/messagebus.py

```diff
@@ -36,11 +36,12 @@
 
 class MessageContext(BaseModel):
     model_config = ConfigDict(extra="allow")
-    session: Optional[SessionContext] = Field(description="Session Data",
-                                              default=None)
+    session: SessionContext = Field(description="Session Data",
+                                              default=SessionContext())
```

Comment:
I assume this is a breaking change, is that going to be an issue for any projects using this?

Reply:
I don't think this will break anything, it actually came up because I found an unsafe reference in the wild where a `None` value broke something.

I can add a validator though in case `None` is explicitly passed to make sure it handles the default value